Client/Frontend/Browser/Browser.swift

```diff
@@ -290,6 +290,18 @@ class Browser: NSObject {
     }
 
     func reload() {
+        if #available(iOS 9.0, *) {
+            webView?.customUserAgent = desktopSite ? UserAgent.desktopUserAgent() : nil
+
+            if let currentItem = webView?.backForwardList.currentItem where currentItem.URL != currentItem.initialURL {
+                // Reload the initial URL to avoid UA specific redirection
+                loadRequest(NSURLRequest(URL: currentItem.initialURL, cachePolicy: .ReloadIgnoringLocalAndRemoteCacheData, timeoutInterval: 60))
+                return
+            }
+        }
+
+        // Reload the current URL. We cannot use loadRequest in this case because it seems to leverage caching which might break UA spoofing.
```

Comment:
I think you can just remove this comment -- it will seem out-of-place outside of the context of this PR.